pxtpy/converter.ts

```diff
@@ -2181,7 +2181,16 @@ namespace pxt.py {
         },
         Subscript: (n: py.Subscript) => {
             if (n.slice.kind == "Index") {
-                unifyTypeOf(n, typeOf(n.value));
```

Comment:
I guess this never worked in the slightest!

Reply:
Actually I see the intent now. It works for strings. I added a case back for strings.